app/views/pages/_contributor_blurb.html.erb

```diff
@@ -12,13 +12,13 @@
     body: <<~HEREDOC
       <div class="contributorModalBody">
         <div class="contributorModalBodyPhoto hideOnSmall">
-          <img class="contributorAvatar" src="#{ProfilePicture.normalize_url(blurb["image"])}">
+          <img class="contributorAvatar" src="#{ProfilePicture.normalize_url(blurb["image"])} alt="Contributor">
```

Comment:
I think these avatars are not important to the context of this modal. So we can make this image purely presentational by using `alt=""`. 

Similar comment for your change below!

Reply:
Sure! Will fix it.